.gitlab-ci.yml

```diff
@@ -62,6 +62,6 @@ publish:
   <<: *image
   stage: publish
   only:
-    - master
+    - branches
```

Comment:
should stay master no?

Reply:
it's a draft because I'm trying to understand how to push it to do it! at the moment it does not work